packages/cloudflare/src/index.ts

```diff
@@ -225,7 +168,8 @@ export default function createIntegration(args?: Options): AstroIntegration {
 								caches: platformProxy.caches,
 								ctx: {
 									waitUntil: (promise: Promise<any>) => platformProxy.ctx.waitUntil(promise),
-									passThroughOnException: () => platformProxy.ctx.passThroughOnException(),
+									// Currently not available: https://developers.cloudflare.com/pages/platform/known-issues/#pages-functions
+									// passThroughOnException: () => platformProxy.ctx.passThroughOnException(),
```

Comment:
We don't need to do it right away, but I am wondering, what if `platformProxy.ctx.passThroughOnException()`, when used in a Pages application, would throw an error or something warning users not to use it?

(I feel like that could be more convenient here instead of hiding the function, since I'd imagine that types would still show the function as present in any way, right?)